labs/lab6/lab6.adoc

```diff
@@ -29,56 +37,311 @@ Password: INSTRUCTOR WILL PROVIDE
 
-The _0-Self-Configure_ job template executed earlier configured a set of templates to automate the process to expand the OpenShift cluster.
+After browsing through the application, change directories to the _alertmanager-tower-bridge-apb_ directory where we will walkthrough the process of creating an Ansible Playbook Bundle.
 
-Using a web browser from the student workstation machine, navigate to the Ansible Tower instance:
+The link:https://access.redhat.com/documentation/en-us/openshift_container_platform/3.9/html-single/ansible_playbook_bundle_development_guide/index[Ansible Playbook Development Guide] provides a detailed overview for developing Ansible Playbook Bundles. The majority of the functionality has already been provided for you.  Take a moment to navigate through the link:https://access.redhat.com/documentation/en-us/openshift_container_platform/3.9/html-single/ansible_playbook_bundle_development_guide/index#apb-devel-writing-ref-directory[folder structure] including the APB spec file (apb.yml) and the directories containing Ansible link:http://docs.ansible.com/ansible/latest/user_guide/playbooks.html[playbooks] and link:https://docs.ansible.com/ansible/devel/user_guide/playbooks_reuse_roles.html[roles].
 
-link:https://tower-<student_id>.labs.sysdeseng.com[https://tower-<student_id>.labs.sysdeseng.com] 
+Build the apb which will execute a Docker build using the _Dockerfile_ located within the _alertmanager-tower-bridge-apb_ directory.
+
+.master$
+[source, bash]
+----
+apb build
+----
+
+A new image containing the APB was created locally. This can be confirmed by listing all of the images on the machine and locating the image called _alertmanager-tower-bridge-apb_.
+
+.master$
+[source, bash]
+----
+docker images
+----
+
+The final step in the Ansible Playbook Bundle creation process is to push the image from the local machine to OpenShift's internal registry. In order to communicate with the registry, an authenticated user to the platform must be used as they contain an OAuth token needed to facilitate the communication.
+
+Login as the "<student_id>-admin" user which has elevated cluster privileges:
+
+.master$
+[source, bash]
+----
+oc login -u <student_id>-admin
+----
+
+Push the APB to the OpenShift integrated registry
+
+.master$
+[source, bash]
+----
+apb push
+----
+
+Confirm the APB is available in the Ansible Service broker by listing all registered APB's.
+
+.master$
+[source, bash]
+----
+apb list
```

Comment:
Is it worth mentioning the pre-work that went in somewhere?  Specifically the whitelisting that was done to enable parsing / searching of different projects and registries.